### ISSUE TITLE
encoding issue while saving subject in metadata

### ISSUE THREAD

**Issue by gaubert** [gaubert/gmvault#200]
Bug due to again an encoding issue when storing email subject in metadata. See an example below

```
[2015-04-26 20:41]:DEBUG:gmvault:metadata info collected: {'X-GM-MSGID': 946600788774435292, 'INTERNALDATE': datetime.datetime(1998, 8, 10, 4, 34, 20), 'X-GM-LABELS': [u'Already Read/Rare/Friends'], 'SEQ': 2043, 'FLAGS': ('$NotJunk', 'NotJunk', '\\Seen'), 'BODY[HEADER.FIELDS (MESSAGE-ID SUBJECT X-GMAIL-RECEIVED)]': 'Message-ID: <722caad5.35ceda9d@aol.com>\r\nSubject: \x86berraschung! F\x9fr Marc\r\n\r\n', 'X-GM-THRID': 946600788774435292}

[2015-04-26 20:41]:ERROR:gmvault:Cannot read file 946600788774435292.meta. Try to fetch the data again
Traceback (most recent call last):
  File "gmv/gmvault.pyc", line 329, in check_email_on_disk
  File "gmv/gmvault_db.pyc", line 622, in unbury_metadata
  File "json/__init__.pyc", line 278, in load
  File "json/__init__.pyc", line 326, in loads
  File "json/decoder.pyc", line 366, in decode
  File "json/decoder.pyc", line 384, in raw_decode
ValueError: No JSON object could be decoded
[2015-04-26 20:41]:DEBUG:gmvault:Get Data for 946600788774435292.
[2015-04-26 20:41]:DEBUG:gmvault:Storing on disk data for 946600788774435292
[2015-04-26 20:41]:CRITICAL:gmv:Error: 'utf8' codec can't decode byte 0x86 in position 0: invalid start byte.

[2015-04-26 20:41]:CRITICAL:gmv:=== Exception traceback ===
[2015-04-26 20:41]:CRITICAL:gmv:Traceback (most recent call last):
  File "gmv/gmv_cmd.pyc", line 741, in run
  File "gmv/gmv_cmd.pyc", line 661, in _sync
  File "gmv/gmvault.pyc", line 607, in sync
  File "gmv/gmvault.pyc", line 570, in _sync_emails
  File "gmv/gmvault.pyc", line 528, in _common_sync
  File "gmv/gmvault.pyc", line 154, in handle_sync_imap_error
UnicodeDecodeError: 'utf8' codec can't decode byte 0x86 in position 0: invalid start byte

[2015-04-26 20:41]:CRITICAL:gmv:=== End of Exception traceback ===
```

**Comment by gaubert** [comment]
adding encoding guessing and conversion to utf-8 prior saving the text in the file.
fixed in 1.9.1

**Comment by gaubert** [comment]
close it now
